lib/api/meeting-data.ts

```
@@ -0,0 +1,47 @@
+/**
+ * Get the meeting data for the current bot. Since it is called from RSCs,
+ * it requires the complete URL + needs the apiKey to be passed
+ * @param apiKey - The API key for the current user
+ * @param bot_id - The bot id
+ * @param include_transcripts - Whether to include transcript data in the response (defaults to true)
+ * @returns The meeting data for the current bot
+ */
+export async function getMeetingData(
+  apiKey: string,
+  bot_id: string,
+  include_transcripts: boolean = true
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)